x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/evaluate_condition.ts

```diff
@@ -45,16 +45,14 @@ export const evaluateCondition = async ({
   compositeSize: number;
   filterQuery?: string;
   lookbackSize?: number;
-  startTime?: number;
+  startedAt: Date;
   logger: Logger;
 }): Promise<Record<string, ConditionResult>> => {
   const { metric, customMetric } = condition;
 
-  const to = startTime ? moment(startTime) : moment();
-
   const timerange = {
-    to: to.valueOf(),
-    from: calcualteFromBasedOnMetric(to, condition, nodeType, metric, customMetric),
+    to: startedAt.valueOf(),
```

Comment:
`startedAt` still confuses me. I would assume `to` refers to the time something `ended`, no? Shouldn't we call it 
```suggestion
    to: endedAt.valueOf(),
```

Reply:
I used `startedAt` to maintain consistency with the Alerting framework.

Reply:
I changed this to `executionTimestamp`, I feel like that's more descriptive of what the timestamp really is. We just happen to be using it as the `to` portion of the time range.